tests/test_nesta_io.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import zipfile
+import shutil
+import pathlib
+import sys
+sys.path.insert(1,'../nesta_ds_utils/')
```

Comment:
Just a note here: we will have to remember to change this if we decide to go for the `src/nesta_ds_utils` layout